### DISCUSSION TITLE
Transformation status not updated automatically

### DISCUSSION THREAD

**Discussion by zhangxiaomei** [DIRACGrid/DIRAC#7938] [Q&A] [answered]
These three transformation tasks have finsihed, but the file status is still in "Assigned".  
![图片](https://github.com/user-attachments/assets/2e63b264-343b-40cc-931c-1e77cd82bd5a)
Is it possible to update them by hand?

**Comment by andresailer** [comment, marked as answer]
[dirac-transformation-cli](https://dirac.readthedocs.io/en/latest/Commands/index.html#dirac-transformation-cli) let's you change statuses by hand for example with the `setFileStatus` command

**Comment by zhangxiaomei** [comment]
ok, I see.  Do you know why this happened? It is possible to avoid it?

**Reply by andresailer** [reply]
No I don't know why this happened. I got https://gitlab.cern.ch/CLICdp/iLCDirac/ILCDIRAC/-/blob/Rel-v35r0/src/ILCDIRAC/ILCTransformationSystem/Agent/FileStatusTransformationAgent.py?ref_type=heads to fix these things automatically if they happen.

**Reply by zhangxiaomei** [reply]
ok, I see. Thank you for the suggestion.

**Reply by chaen** [reply]
Could it be https://github.com/DIRACGrid/DIRAC/issues/7116 ?
